config/bin_version_strings.cfg

```diff
@@ -284,6 +284,7 @@ ipset;;unknown;"^ipset\ v[0-9](\.[0-9]+)+?$";"sed -r 's/ipset\ v([0-9](\.[0-9]+)
 ipset;;unknown;"ipset\ v[0-9](\.[0-9]+)+?,\ protocol\ version:\ [0-9]";"sed -r 's/ipset\ v([0-9](\.[0-9]+)+?).*/ipset:\1/'";
 iptables;;gpl;"iptables\ v[0-9](\.[0-9]+)+?([a-z])?$";"sed -r 's/iptables\ v([0-9](\.[0-9]+)+?([a-z])?)$/iptables:\1/'";
 iptables;;gpl;"iptables\ v[0-9](\.[0-9]+)+?([a-z])?";"sed -r 's/iptables\ v([0-9](\.[0-9]+)+?([a-z])?).*/iptables:\1/'";
+iptables;multi_grep;gpl;'"can't initialize iptables table `%s': %s"&&"^[0-9](\.[0-9]+)?+$"';"sed -r 's/([0-9](\.[0-9]+)+?)$/iptables:\1/'";
```

Comment:
can we use ^and $ in the first identifier?

Reply:
Yes added